pkg/query/flamegraph_arrow.go

```diff
@@ -1033,7 +1065,9 @@ func (fb *flamegraphBuilder) prepareNewRecord() error {
 // It adds the children to the children column and the labels intersection to the labels column.
 // Finally, it assembles all columns from the builders into an arrow record.
 func (fb *flamegraphBuilder) NewRecord() (arrow.Record, error) {
-	cleanupArrs := make([]arrow.Array, 0, 18+(2*len(fb.builderLabelFields)))
+	const numCols = 17
```

Comment:
why not leave 18, but instead make it 17+1? 🤔

Reply:
Because the 17 is used a couple of times below to initialize `arrays` and `cleanupArrays` . 
To be honest, I'm not totally sure why we initialized this array with 18 compared to all other instances.